env0/resource_project_policy.go

```diff
@@ -3,13 +3,18 @@ package env0
 import (
 	"context"
 	"errors"
+	"fmt"
+	"strings"
 
 	"github.com/env0/terraform-provider-env0/client"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 )
 
 func resourcePolicy() *schema.Resource {
+	allowedProjectTtlValues := append(allowedTtlValues, "inherit")
```

Comment:
I decided to just add "inherit" and not "null".
The reason for not adding "null" - it's not available in the UI.